andrew-mason/scripts/articleView.js

```diff
@@ -87,17 +76,17 @@ articleView.handleCategoryFilter = function() {
 articleView.handleMainNav = function() {
   // TODO: Add an event handler to .main-nav elements that will power the Tabs feature.
   $('.tab').on('click', function() {
-    console.log(this);
+
     $('.tab-content').hide();
-    $(``).show();
+    $(`section[id="${$(this).attr('data-content')}"]`).fadeIn(150);
 
```

Comment:
here we also need to re hide the template class. When we force the first click when the page loads, the template is left visible